src/Adapter/Import/ImportDataMatchConfiguration.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Adapter\Import;
```

Comment:
same here, why its in `Adapter` if theres no legacy dependenices? :thinking:

Reply:
no idea about this one, I'll move it to `Core`.